engine/src/main/java/org/hibernate/search/engine/metadata/impl/SortableFieldMetadata.java

```diff
@@ -21,6 +21,7 @@ private SortableFieldMetadata(String fieldName) {
 
 	/**
 	 * The name of the field whose field bridge will be used to determine the value for the doc value field.
+	 * @return the name of the field
```

Comment:
Agreed, the tool output is horrible, although it seems to list the actual errors at the end in recent JDK versions (at least in some cases, sometimes it's missing :/). Still I think we should not sacrifice readability of the generated docs for the sake of making the tool happy.

> Maybe the following is better?

No, IMO not. The method description (or its first sentence) is shown in the "method summary" block, whereas the return value description is only visible in the details block.